go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/containers/image/v5
 
-go 1.22.5
+go 1.22.6
+
+toolchain go1.22.7
```

Comment:
DONOT MERGE this way.

Reply:
Updated to also include the same comment as in https://github.com/containers/storage/pull/2086 .